ddtrace/appsec/_iast/taint_sinks/_base.py

```diff
@@ -91,6 +92,9 @@ def _prepare_report(cls, span, vulnerability_type, evidence, file_name, line_num
     @classmethod
     def report(cls, evidence_value: Text = "", dialect: Optional[Text] = None) -> None:
         """Build a IastSpanReporter instance to report it in the `AppSecIastSpanProcessor` as a string JSON"""
+        if not _is_iast_enabled():
+            return None
```

Comment:
Do we still need to move this code from the processor/on_span_start to here? Maybe we can do this in a different PR? I'd rather have them separate in case we need to revert